debian/rules

```diff
@@ -3,26 +3,19 @@
 include /usr/share/postgresql-common/pgxs_debian_control.mk
 
 override_dh_auto_build:
-	+make PG_CONFIG=/usr/lib/postgresql/11/bin/pg_config
-	+make
-	+cd src/monitor && pg_buildext build build-%v
+	+pg_buildext build $(shell pwd)/src/bin/citusha build-%v
```

Comment:
What about the alternative packaging which consists of having a separate `citusha` and `postgresql-X-citusha` packages from a single `debian/control.in` file. In debian you can have multiple binary packages built from the same source package. I think that would be the best way to deal with the problem.

Reply:
done,

given that the extension was already called `citus-ha` I went for the same in the new package.